x-pack/packages/ml/agg_utils/src/types.ts

```diff
@@ -88,77 +88,77 @@ export interface HistogramField {
 }
 
 /**
- * Enumeration of significant term types.
+ * Enumeration of significant item types.
  */
-export const SIGNIFICANT_TERM_TYPE = {
+export const SIGNIFICANT_ITEM_TYPE = {
   KEYWORD: 'keyword',
   LOG_PATTERN: 'log_pattern',
 } as const;
 
 /**
- * Type for significant term type keys.
+ * Type for significant item type keys.
  */
-type SignificantTermTypeKeys = keyof typeof SIGNIFICANT_TERM_TYPE;
+type SignificantItemTypeKeys = keyof typeof SIGNIFICANT_ITEM_TYPE;
 
 /**
- * Represents the type of significant term as determined by the SIGNIFICANT_TERM_TYPE enumeration.
+ * Represents the type of significant item as determined by the SIGNIFICANT_ITEM_TYPE enumeration.
  */
-export type SignificantTermType = typeof SIGNIFICANT_TERM_TYPE[SignificantTermTypeKeys];
+export type SignificantItemType = typeof SIGNIFICANT_ITEM_TYPE[SignificantItemTypeKeys];
 
 /**
- * Represents significant term metadata for a field/value pair.
+ * Represents significant item metadata for a field/value pair.
  * This interface is used as a custom type within Log Rate Analysis
  * for a p-value based variant, not related to the generic
  * significant terms aggregation type.
  *
  * @interface
  * @extends FieldValuePair
  */
-export interface SignificantTerm extends FieldValuePair {
-  /** The key associated with the significant term. */
+export interface SignificantItem extends FieldValuePair {
+  /** The key associated with the significant item. */
   key: string;
 
-  /** The type of the significant term. */
-  type: SignificantTermType;
+  /** The type of the significant item. */
+  type: SignificantItemType;
 
-  /** The document count for the significant term. */
+  /** The document count for the significant item. */
   doc_count: number;
 
-  /** The background count for the significant term. */
+  /** The background count for the significant item. */
   bg_count: number;
 
-  /** The total document count for all terms. */
+  /** The total document count for all items. */
   total_doc_count: number;
 
-  /** The total background count for all terms. */
+  /** The total background count for all items. */
   total_bg_count: number;
 
-  /** The score associated with the significant term. */
+  /** The score associated with the significant item. */
   score: number;
 
-  /** The p-value for the significant term, or null if not available. */
+  /** The p-value for the significant item, or null if not available. */
   pValue: number | null;
 
-  /** The normalized score for the significant term. */
+  /** The normalized score for the significant item. */
   normalizedScore: number;
 
-  /** An optional histogram of significant term items. */
-  histogram?: SignificantTermHistogramItem[];
+  /** An optional histogram of significant item items. */
```

Comment:
nit: there is an extra `item`

Reply:
Thanks for spotting this, updated in [8437d13](https://github.com/elastic/kibana/pull/169756/commits/8437d13ef3664124be52e71d218b0039cac5405b).